schemas/src/elements/TableCell.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+    "description": "Represents a cell within a row of a Table element.",
+    "extends": "Container",
```

Comment:
Container defines VerticalContentAlignment to be a required property with default "Top". This should now be changed to optional with no default, right? Same for horizontal alignment on TextBlock.

Reply:
Ah correct, will change

Reply:
Fixed